### ISSUE TITLE
Accessibility - Hamburger menu is missing focus indicator when tabbing through fields

### ISSUE THREAD

**Issue by maur1** [DependencyTrack/frontend#1202]
### Current Behavior

Hamburger menu is missing focus indicator when tabbing through fields

### Proposed Behavior

Add CSS to ensure the menu button has a visible focus state

### Checklist

- [x] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/master/CONTRIBUTING.md#filing-issues)
- [x] I have checked the [existing issues](https://github.com/DependencyTrack/frontend/issues) for whether this enhancement was already requested
